lib/apiservers/portlayer/swagger.json

```diff
@@ -2095,6 +2204,17 @@
 				}
 			}
 		},
+		"KeyValue": {
+			"type": "object",
+			"properties": {
+				"Key": {
+					"type": "string"
+				},
+				"Value":{
+					"type": "string"
```

Comment:
based on your previous comment about "limiting" the k/v -- I understand the desire for a change...